### DISCUSSION TITLE
DAI 2024-2025 - My picocli project - Nathan Tschantz

### DISCUSSION THREAD

**Discussion by TschantzN** [heig-vd-dai-course/heig-vd-dai-course#307] [Show and tell]
https://github.com/TschantzN/04-java-intellij-idea-and-maven

**Comment by ludelafo** [comment]
Everything seems fine for me except the `pom.xml`. It seems you have removed the initial properties from the following lines:

```xml
<properties>

</properties>
```

You must keep the initial values to ensure Maven can encode the files correctly (more on this in the next course) and check for the right Java version.
